arbeitszeit/use_cases/approve_plan.py

```diff
@@ -31,7 +31,7 @@ def approve_plan(self, request: Request) -> Response:
         planner = self.database_gateway.get_companies().with_id(plan.planner).first()
         assert planner
         if plan.is_approved:
-            return self.Response(is_approved=False)
+            return self.Response(is_plan_approved=False)
```

Comment:
Sorry!  Is this right?  If plan.is_approved, then return a response where is_plan_approved is False?  It seems wrong.

Reply:
yes, it is because the plan was already approved before, so this case is more like a theoretical one, it is basically not possible due to the UI. afaics, it is just about which notification would be shown in the UI, i.e. 
"Plan was approved successfully"
or
"Plan approval failed"
In case the plan was already approved, you would see "Plan approval failed" then, but as said, this case is prevented because once an accountant approved/rejects a plan, it will disappear from the only list accountants have.